packages/search/src/ago/helpers/aggs/downloadable.ts

```diff
@@ -0,0 +1,68 @@
+import { ISearchResult, IItem } from "@esri/arcgis-rest-portal";
+
+// implements the 'downloadable' facet from AGO results. V3 datasets have this property computed
+// during the harvesting process but AGO results need this result computed at runtime
+export function downloadable(response: ISearchResult<IItem>): any {
+  // Get counts of true and false for downloadable facet
+  // i.e. { true: 10, false: 15 }
+  return response.results.reduce(
+    (formattedAggs: any, result: IItem) => {
+      if (
+        result.typeKeywords.indexOf("Data") > -1 ||
+        downloadableTypes.indexOf(result.type) > -1
+      ) {
+        formattedAggs.downloadable.true++;
+      } else {
+        formattedAggs.downloadable.false++;
+      }
+      return formattedAggs;
+    },
+    { downloadable: { true: 0, false: 0 } }
+  );
+}
+
+// eligible types are listed here: http://doc.arcgis.com/en/arcgis-online/reference/supported-items.htm
+const downloadableTypes = [
```

Comment:
given what we already started in `@esri/hub-common`, would it make more sense to export this type from there?